packages/core/helper-plugin/src/features/StrapiApp.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+
+import { TranslationMessage } from 'types';
+
+interface Permission {
+  action: string;
+  subject: string | null;
+}
```

Comment:
imo this should come from `@strapi/permissions` no?

Reply:
Nice, thanks!